includes/scripts.php

```diff
@@ -104,12 +27,10 @@ function d7_enqueue_jquery() {
 		'currentTheme' => get_current_theme(),
 		'url' => get_bloginfo('url'),
 	);
-	wp_localize_script('basic', 'D7WP', $wp_object);
 
+	wp_localize_script('basic', 'D7WP', $wp_object);
 
-	// For comment reply form
```

Comment:
In general, I like to comment very sparsely, so I wouldn't commit a comment like that.  I think it should stay in PR though since the things in PR are distinct from that.

If you'd like, I can make a PR to remove comments that are of the form
```
// does the thing
function doThing(){}
thing = ['something'] // make thing
```